### PR TITLE
Upgrade to sailor 2.6.18

### PR DESCRIPTION
**Pull request by denyshld**
Annual audit of the component code to check if it exposes a sensitive data in the logs

### CODE REVIEW

File: lib/actions/lookupObject.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ exports.process = async function ProcessAction(msg, cfg) {
     foundObject = await instance.makeRequest(`${encodeURIComponent(cfg.module)}/${msg.body.id}`, 'GET');
   } catch (err) {
     this.logger.error('Error during lookup object occurred');
-    this.logger.trace(err);
+    this.logger.trace('Error during lookup object occurred');
```
Comment (shulkaolka) [inline lib/actions/lookupObject.js:26]:
```suggestion
```

File: lib/actions/lookupObject.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ exports.process = async function ProcessAction(msg, cfg) {
       return;
     }
     const err = new Error('The object has not been found');
-    this.logger.error(err);
+    this.logger.error('The object has not been found');
```
Comment (shulkaolka) [inline lib/actions/lookupObject.js:38]:
```suggestion
    this.logger.error(err);
```

**Review by shulkaolka — CHANGES_REQUESTED** [review]

File: lib/actions/lookupObject.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ exports.process = async function ProcessAction(msg, cfg) {
       return;
     }
     const err = new Error('No ID provided');
-    this.logger.error(err);
+    this.logger.error('No ID provided');
```
[the remark this answers is not in the document]
Comment (denyshld) [reply lib/actions/lookupObject.js:14]: Need to keep refactoring style

File: lib/actions/lookupObject.js (hunk 2 above)
Comment (denyshld) [reply lib/actions/lookupObject.js:38]: Need to keep refactoring style

**Review by shulkaolka — APPROVED** [review]

**Pull request merged**